images/base/files/usr/local/bin/entrypoint

```diff
@@ -18,13 +18,24 @@ set -o errexit
 set -o nounset
 set -o pipefail
 
+# logging helpers
+log_info() {
+  echo "INFO: $1" >&2
```

Comment:
we were not logging info to stderr, is this change on purpose?

Reply:
We mostly were (which is correct for diagnostic messages), but not always